blueflood-elasticsearch/src/main/resources/index_settings.json

```diff
@@ -0,0 +1,27 @@
+{
+    "index": {
+        "analysis": {
+            "analyzer": {
+                "prefix-test-analyzer": {
+                    "filter": "dotted",
+                    "tokenizer": "prefix-test-tokenizer",
+                    "type": "custom"
+                }
+            },
+            "filter": {
+                "dotted": {
+                    "patterns": [
+                        "([^.]+)"
+                    ],
+                    "type": "pattern_capture"
+                }
+            },
+            "tokenizer": {
+                "prefix-test-tokenizer": {
+                    "delimiter": ".",
+                    "type": "path_hierarchy"
+                }
+            }
+        }
+    }
+}
```

Comment:
I know you may not be able to document this json, but could you find a place to write one paragraph on what this actually does, so people don't have to google it.  You could just include these pointers:
http://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-pattern-capture-tokenfilter.html

http://www.elastic.co/guide/en/elasticsearch/reference/0.90/analysis-pathhierarchy-tokenizer.html